.github/workflows/teardown-frontend-dev.yml

```
@@ -0,0 +1,332 @@
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Generate environment name
+        id: env-name
+        run: |
+          ENV_NAME="pr-${{ github.event.inputs.pr_number }}"
+          echo "env-name=${ENV_NAME}" >> $GITHUB_OUTPUT
+          echo "🌍 Environment name: ${ENV_NAME}"
+
+      - name: Generate Amplify app info
+        id: amplify-info
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:48: Double quote to prevent globbing and word splitting

(shellcheck)